Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM maven:3.5-jdk-8-alpine as maven
```

Comment:
This docker config is temporary. Need to find out on how to pull and run test runner app on e2 micro vm.